icons/etcd.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Etcd</title><path d="M10.985 10.67A1.565 1.62 0 1 1 9.42 9.051a1.566 1.62 0 0 1 1.565 1.62zm2.024 0a1.565 1.62 0 1 0 1.565-1.619 1.564 1.62 0 0 0-1.565 1.62zm10.653 1.758a4.295 4.446 0 0 1-.346.014 4.517 4.676 0 0 1-1.986-.479 18.449 19.096 0 0 0 .267-3.638 18.185 18.823 0 0 0-2.274-2.79 4.52 4.678 0 0 1 1.603-1.777l.294-.189-.23-.268A11.977 12.398 0 0 0 16.808.143L16.488 0l-.08.347a4.506 4.665 0 0 1-1.134 2.131 18.19 18.829 0 0 0-3.277-1.397 18.127 18.763 0 0 0-3.273 1.395A4.495 4.653 0 0 1 7.594.35L7.512.001l-.317.143a12.092 12.516 0 0 0-4.182 3.157l-.23.268.294.189a4.512 4.67 0 0 1 1.6 1.767 18.323 18.966 0 0 0-2.27 2.78 18.436 19.083 0 0 0 .26 3.662 4.505 4.663 0 0 1-1.976.474 4.224 4.372 0 0 1-.346-.013L0 12.4l.032.356a11.904 12.323 0 0 0 1.61 5.097l.174.309.263-.232a4.502 4.66 0 0 1 2.131-1.033 18.29 18.932 0 0 0 1.824 3.076 18.474 19.122 0 0 0 3.457.88 4.493 4.65 0 0 1-.287 2.442l-.132.331.338.077A12.049 12.472 0 0 0 12 24l2.59-.297.339-.077-.131-.331a4.487 4.645 0 0 1-.287-2.444 18.477 19.125 0 0 0 3.443-.878 18.209 18.848 0 0 0 1.826-3.078 4.51 4.67 0 0 1 2.143 1.034l.263.23.175-.306a11.877 12.294 0 0 0 1.607-5.097L24 12.4zm-7.958 4.356a13.982 14.473 0 0 1-7.417 0 14.19 14.688 0 0 1-2.256-7.258 14.119 14.614 0 0 1 2.687-2.649 14.334 14.836 0 0 1 3.28-1.846 14.378 14.883 0 0 1 3.27 1.841 14.227 14.726 0 0 1 2.7 2.667 14.294 14.795 0 0 1-.675 3.78 14.366 14.87 0 0 1-1.59 3.465z"/></svg>
```

Comment:
I assume you used the SVG in the top left of the website as the source, correct?

If so, it seems something didn't go quite right, see my comparison below. It looks like you resized the SVG to exactly 24x24 pixels. However, the icon isn't a perfect square so as a result the SVG is now distorted. Could you try this again but this time preserving the aspect ratio of the original?

![image](https://user-images.githubusercontent.com/3742559/127745907-37d85624-391d-40f8-a30e-a45aaadcec87.png)

Reply:
corrected already

Reply:
Perfect, nice work! :100: